cmd/contour/shutdownmanager.go

```diff
@@ -239,6 +245,7 @@ func registerShutdown(cmd *kingpin.CmdClause, log logrus.FieldLogger) (*kingpin.
 	shutdown := cmd.Command("shutdown", "Initiate an shutdown sequence which configures Envoy to begin draining connections.")
 	shutdown.Flag("check-interval", "Time to poll Envoy for open connections.").DurationVar(&ctx.checkInterval)
 	shutdown.Flag("check-delay", "Time wait before polling Envoy for open connections.").Default("60s").DurationVar(&ctx.checkDelay)
+	shutdown.Flag("drain-delay", "Time wait before draining Envoy connections.").Default("0s").DurationVar(&ctx.drainDelay)
```

Comment:
nit: "Time to wait..." (it'd be great if you could update the line just above as well since it has the same typo)